base/VulkanDebug.cpp

```diff
@@ -17,7 +17,7 @@ namespace vks
 		PFN_vkDestroyDebugUtilsMessengerEXT vkDestroyDebugUtilsMessengerEXT;
 		VkDebugUtilsMessengerEXT debugUtilsMessenger;
 
-		VKAPI_ATTR VkBool32 VKAPI_CALL debugUtilsMessengerCallback(
```

Comment:
Thanks for spotting this!

Reply:
Weird. I honestly didn't mean to make that change.  I think I changed both to `messageCallback` and then thought better of it and tried to rename it back but started with the header and misremembered the name.